### ISSUE TITLE
Viewport - Unexpected behavior for rotation manipulator

### ISSUE THREAD

**Issue by AMZN-daimini** [o3de/o3de#2728]
**Describe the bug**
When using the rotation manipulator, I expect the object to keep on rotating if I drag it and keep moving the mouse in one direction (for example, left for clockwise). Instead, the entity will only rotate about 90 degrees, then stall. The manipulator expects the user to circle the cursor around the entity, which I don't think is intuitive.

**To Reproduce**
1. Create an entity
2. Add a Mesh (not necessary, but useful for visual feedback)
3. Press 2 or select the rotate icon in the viewport
4. click on the manipulator from the bottom and drag it left to the edge of the screen (and beyond)

**Expected behavior**
The object keeps rotating clockwise as long as the user keeps dragging left.

**Actual behavior**
The rotation stalls after about 90 degrees

**Screenshots**
![rotationManipulator](https://user-images.githubusercontent.com/82231674/127914822-eb46569b-64e5-44b0-8d89-2f0171cfee65.gif)

**Comment by hultonha** [comment]
I think this could be dramatically improved by better visualisation/feedback when it comes to the drag (e.g. render the grab point and interior angle). The design you proposed has drawbacks too and can cause weird behaviours. Ideally we'd have the option to pick a different kind of manipulator for rotation to map to this form (one that isn't a circle for example)

**Comment by guillaume-haerinck** [comment]
I agree that the feedback is good enough to explain that the mouse has to be rotated around the object. If further down the road we want to support another mouse move mode, that should be a separate feature request
